core/jsfl/libraries/objects/XML.jsfl

```diff
@@ -37,7 +37,7 @@
 		 * @type {RegExp}	type, attr, operator, value
 		 * @ignore
 		 */
-		var rxFilter		= /([@#\.])([\w_-:]+)([\^\$!=<>]+)?([^"'\(\)]+)?/;
+		var rxFilter		= /([@#\.])([\w:-_]+)([\^\$!=<>]+)?([^"'\(\)]+)?/;
```

Comment:
Thanks Hays! Yes, I've solved this same bug in other projects since. My workaround these days is to put the dash character first in a character class group.